### PR TITLE
Remove trailing spaces from EN documents(#16742)

### PR DESCRIPTION
**Pull request by furukawa3**
This PR removes trailing spaces from en doc.  Following command are used.
```
sed -i 's/[ ]*$//' $(find content/en -type f -not -name "*.png" -not -name "*.jpg" -not -name "*.svg" -not -name "*.gif")
```

### CODE REVIEW

**Comment by zacharysarah** [comment]
@furukawa3 One request: please omit changes to files in the `content/en/blog/` path.

**Comment by furukawa3** [comment]
> @furukawa3 One request: please omit changes to files in the `content/en/blog/` path.

Sure.  I'll update it.

**Comment by kbhawkey** [comment]
@furukawa3 , What do you think about omitting the `content/en/case-studies` files?

**Comment by furukawa3** [comment]
> case-studies
Hi.  I think it's OK to omit theses files if there is at least a reason to put trailing spaces.

**Comment by furukawa3** [comment]
> > case-studies
> 
> Hi. I think it's OK to omit theses files if there is at least a reason to put trailing spaces.

@kbhawkey Could you reply to my comment?  Do you think it's better to omit ``content/en/case-studies`` files ??

**Comment by kbhawkey** [comment]
> 
> 
> > case-studies
> 
> Hi. I think it's OK to omit theses files if there is at least a reason to put trailing spaces.

hello @furukawa3 . Sorry for the late reply. My area is in the middle of a storm and we were without electricity for some time! 
Since there is only one `case studies` html file, it does not make sense to omit.
While I think it is nice to have the extra spaces and new lines cleaned up, I _think_ that this kind of far reaching merge will cause a number of back merges. I wonder if the Markdown parser eliminates the extra spaces when generating the Html?

**Comment by furukawa3** [comment]
> hello @furukawa3 . Sorry for the late reply. My area is in the middle of a storm and we were without electricity for some time!
> Since there is only one `case studies` html file, it does not make sense to omit.
> While I think it is nice to have the extra spaces and new lines cleaned up, I _think_ that this kind of far reaching merge will cause a number of back merges. I wonder if the Markdown parser eliminates the extra spaces when generating the Html?

I'm so sorry I was late reply..

> I wonder if the Markdown parser eliminates the extra spaces when generating the Html?

I checked [case-study]() page and found Markdown parser eliminates the extra spaces.  Therefore, I want to remove unnecessary trailing spaces.

File: content/en/case-studies/northwestern-mutual/index.html [hunk 1]
```diff
@@ -22,7 +22,7 @@ <h1> CASE STUDY:<img src="/images/northwestern_logo.png" style="margin-bottom:-1
...
+    In the spring of 2015, Northwestern Mutual acquired a fintech startup, LearnVest, and decided to take "Northwestern Mutual’s leading products and services and meld it with LearnVest’s digital experience and innovative financial planning platform," says Brad Williams, Director of Engineering for Client Experience, Northwestern Mutual. The company’s existing infrastructure had been optimized for batch workflows hosted on on-prem networks; deployments were very traditional, focused on following a process instead of providing deployment agility. "We had to build a platform that was elastically scalable, but also much more responsive, so we could quickly get data to the client website so our end-customers have the experience they expect," says Williams.
```
Comment (kbhawkey) [inline content/en/case-studies/northwestern-mutual/index.html]: hello @furukawa3 . I'll try and review each file.
Generally, an across-the-file-system merge requires a lot of back merging for all.
Also, the rendering of each changed file should be checked.
I wonder if these changes should wait until after the 1.17 release.
Line 25: Is there a newline or space removed? I don't see a change.

File: content/en/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ The aggregation layer allows Kubernetes to be extended with additional APIs, bey
...
 
-The aggregation layer runs in-process with the kube-apiserver. Until an extension resource is registered, the aggregation layer will do nothing. To register an API, users must add an APIService object, which "claims" the URL path in the Kubernetes API. At that point, the aggregation layer will proxy anything sent to that API path (e.g. /apis/myextension.mycompany.io/v1/…) to the registered APIService. 
+The aggregation layer runs in-process with the kube-apiserver. Until an extension resource is registered, the aggregation layer will do nothing. To register an API, users must add an APIService object, which "claims" the URL path in the Kubernetes API. At that point, the aggregation layer will proxy anything sent to that API path (e.g. /apis/myextension.mycompany.io/v1/…) to the registered APIService.
```
Comment (kbhawkey) [inline content/en/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md]: @furukawa3 , do you see a change here?

File: content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md [hunk 3]
```diff
@@ -45,28 +45,28 @@ kind: Pod
 ...
 spec:
 	schedulerName: poseidon
```
Comment (kbhawkey) [inline content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md]: @furukawa3 , if you have time, would you clean up line 47 (tab)?

File: content/en/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 4]
```diff
@@ -185,18 +185,18 @@ Once Pod is assigned to a node by scheduler, kubelet starts creating containers
        Reason:       ErrImagePull
 	  ...
    ```
-
```
Comment (kbhawkey) [inline content/en/docs/concepts/workloads/pods/pod-lifecycle.md]: @furukawa3 , Line 186, are there two tabs that could be removed and replaced with spaces?

File: content/en/docs/reference/command-line-tools-reference/kubelet.md [hunk 5]
```diff
@@ -42,17 +42,17 @@ kubelet [flags]
     <tr>
       <td></td><td style="line-height: 130%; word-wrap: break-word;">If true, adds the file directory to the header</td>
     </tr>
-
```
Comment (kbhawkey) [inline content/en/docs/reference/command-line-tools-reference/kubelet.md]: @furukawa3 , this file is auto-generated. I would not bother to remove the white space in this file.

File: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_kubelet_config_download.md [hunk 6]
```diff
@@ -15,7 +15,7 @@ kubeadm alpha kubelet config download [flags]
 ```
   # Download the kubelet configuration from the ConfigMap in the cluster. Autodetect the kubelet version.
   kubeadm alpha phase kubelet config download
-  
+
```
Comment (kbhawkey) [inline content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_kubelet_config_download.md]: @furukawa3 , I would not bother to clean up this file and all files found under the directory:
en/docs/reference/setup-tools/kubeadm/generated/
These files are auto-generated.

File: content/en/docs/setup/release/notes.md [hunk 7]
```diff
@@ -942,8 +942,8 @@ filename | sha512 hash
 * Make kubectl get --ignore-not-found continue processing when encountering error.  ([#82120](https://github.com/kubernetes/kubernetes/pull/82120), [@soltysh](https://github.com/soltysh))
 * Dual stack services (Phase II of IPv6DualStack feature) are enabled via the IPVS proxier. iptables proxier does not support dualstack yet. Dualstack iptables proxier is WIP and should catchup soon. ([#82091](https://github.com/kubernetes/kubernetes/pull/82091), [@khenidak](https://github.com/khenidak))
     * to enable, kube-proxy must be have the following flags:
-    * --proxy-mode=ipvs
```
Comment (kbhawkey) [inline content/en/docs/setup/release/notes.md]: @furukawa3 , I would exclude this file. This file is auto-generated.

File: content/en/case-studies/northwestern-mutual/index.html (hunk 1 above)
Comment (furukawa3) [reply content/en/case-studies/northwestern-mutual/index.html]: Hi @kbhawkey ,  I see.  it's OK to wait until 1.17.  I'll rebase this patch after that.

> Line 25: Is there a newline or space removed? I don't see a change.

I removed a trailing space.  You can check it by copy&paste into other editor (e.g. Vim)  :)

File: content/en/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md (hunk 2 above)
Comment (furukawa3) [reply content/en/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md]: ditto Line:25.   I removed a trailing space.  You can check by copy&paste into other editor as well.

File: content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md (hunk 3 above)
Comment (furukawa3) [reply content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md]: +1 !!  I wanted to cleanup all of  *tab* space :)  However, my other patches were only removing trailing spaces and already merged...  (e.g. FR, ES docs)  So, is it OK to push as an another issue and patches to cleanup *tab* space ??

File: content/en/docs/concepts/workloads/pods/pod-lifecycle.md (hunk 4 above)
Comment (furukawa3) [reply content/en/docs/concepts/workloads/pods/pod-lifecycle.md]: ditto https://github.com/kubernetes/website/pull/16787#discussion_r339849626
I'd like to replace them :)

File: content/en/docs/reference/command-line-tools-reference/kubelet.md (hunk 5 above)
Comment (furukawa3) [reply content/en/docs/reference/command-line-tools-reference/kubelet.md]: I see.  So, is it better to omit  under *en/docs/reference/*  files ??

File: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_kubelet_config_download.md (hunk 6 above)
Comment (furukawa3) [reply content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_kubelet_config_download.md]: I see!!!  Thanks.

File: content/en/docs/setup/release/notes.md (hunk 7 above)
Comment (furukawa3) [reply content/en/docs/setup/release/notes.md]: I see.  Thanks.

File: content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md (hunk 3 above)
Comment (furukawa3) [reply content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md]: Please lemme push an another patch to cleanup tab space if you feel it makes sense.

**Comment by Rajakavitha1** [comment]
@kbhawkey could you please review the changes and state your approval?

**Comment by kbhawkey** [comment]
Helllo @Rajakavitha1 , I will reread the commits again, though, would prefer for this PR to wait until the release is completed.

**Comment by sftim** [comment]
Hi @furukawa3 

Would you be willing to rebase this PR against _master_?

**Comment by zacharysarah** [comment]
@furukawa3 Feel free to `/reopen` this PR when you're ready to rebase.

In the future, it's helpful to break this kind of work down into smaller PRs. Normally I recommend bundling similar work into a single PR, but 100+ files is a lot to review. 😓

**Comment by furukawa3** [comment]
I see.  Thanks zacharysarah .  I'll rebase and split more smaller PRs.

**Pull request closed without merging**